client.go

```diff
@@ -353,6 +354,26 @@ func (c *ovndb) reconnect() {
 	}()
 }
 
+// colToField uses reflection to convert a value from ovsdb to a provided struct field
+func (odbi *ovndb) colToField(column string, from interface{}, to reflect.Value) error {
+	schema := odbi.GetSchema()
+	tableSchema, ok := schema.Tables[TableACL]
```

Comment:
Shouldn't "Table" be a parameter to this function?

Reply:
Yes hence the `WIP` 😂 
I tried this code out on the ACL resource and forgot to factor that out, good catch